### PR TITLE
[7] Create a component that takes in the user input

### PR DESCRIPTION
**Pull request by oescousse**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines. For the guidelines check the README.
- [ ] Tests for the changes have been added (for bug fixes / features). As we discussed this is optional but encouraged.

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
adds a user input component to the app

* **What is the current behavior?** (You can also link to an open issue here)
allows the user to input a zip code as their search

* **What is the new behavior (if this is a feature change)?**
N/A

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

File: src/components/UserInput/UserInput.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import {
+    Container, Col, Form,
+    FormGroup, Label, Input,
+    Button,
+  } from 'reactstrap';
```
Comment (RyanFriedman36) [inline src/components/UserInput/UserInput.js:6]: It looks like you forgot to add a 'Go' or 'Search' button

File: src/components/UserInput/UserInput.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import './UserInput.css';
+
+class UserInput extends Component {
+    render() {
```
Comment (RyanFriedman36) [inline src/components/UserInput/UserInput.js:10]: I had to add 'export' here to get this to work on my computer

File: src/components/UserInput/UserInput.js (hunk 1 above)
Comment (oescousse) [reply src/components/UserInput/UserInput.js:6]: oops you are very correct

File: src/components/UserInput/UserInput.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
+.App {
```
Comment (RyanFriedman36) [inline src/components/UserInput/UserInput.css:1]: I think you want .UserInput here

File: src/components/UserInput/UserInput.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+                            <Label>Zip Code:</Label>
+                            <Input
+                            type="number"
+                            name="number"
+                            id="exampleNumber"
+                            placeholder="00000"
+                            />
+                        </FormGroup>
+                    </Col>
+                    <Button outline color="blue">Search</Button>
```
Comment (RyanFriedman36) [inline src/components/UserInput/UserInput.js:26]: I'm not sure if reactstrap as built in outlines...maybe just go with color="primary" which is blue or color="secondary" which is grey

**Comment by csjoblinksreddit** [comment]
Awesome! Way to go Olivia. Couple of things before I merge this. Is there a way to limit the users to only 5 characters since a zip code at most can be five characters. Also, can you change the name of the pr to include the ticket that you are working on. Other than that great work!

**Pull request merged**